src/TwigComponent/src/Twig/ComponentRuntime.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\TwigComponent\Twig;
+
+use Symfony\UX\TwigComponent\ComponentRenderer;
+use Symfony\UX\TwigComponent\Event\PreRenderEvent;
+use Twig\Extension\RuntimeExtensionInterface;
+
+/**
+ * @author Kevin Bond <kevinbond@gmail.com>
+ * @author Simon André <smn.andre@gmail.com>
+ *
+ * @internal
+ */
+final class ComponentRuntime implements RuntimeExtensionInterface
```

Comment:
(This interface isn't required - it's only for autoconfiguration IIRC)

Reply:
Did not know, thanks